### PR TITLE
Inspect Android and iOS solutions

### PR DESCRIPTION
**Pull request by miterosan**

### CODE REVIEW

File: build/build.cake [hunk 1]
```diff
@@ -20,6 +20,8 @@ var tempDirectory = new DirectoryPath("temp");
 var artifactsDirectory = rootDirectory.Combine("artifacts");
 
 var solution = rootDirectory.CombineWithFilePath("osu-framework.sln");
+var androidSolution = rootDirectory.CombineWithFilePath("osu-framework.sln");
```
Comment (peppy) [inline build/build.cake]: oops?

**Comment by peppy** [comment]
@miterosan did you test this further? would be nice to get running, but appveyor is still not approving of it.

**Comment by swoolcock** [comment]
@miterosan is this PR still required?

**Comment by peppy** [comment]
Looks like it is, the build script is only inspecting desktop still.

**Pull request closed without merging**